src/Factories/FunFunFactory.php

```diff
@@ -1283,4 +1292,18 @@ public function newPageNotFoundHTMLPresenter(): PageNotFoundPresenter {
 		return new PageNotFoundPresenter( $this->getLayoutTemplate( '404message.html.twig' ) );
 	}
 
+	public function setServerSideTracker( ServerSideTracker $tracker ) {
+		$this->pimple['server_side_tracker'] = function () use ( $tracker )  {
+			return $tracker;
+		};
+	}
```

Comment:
Technically, assigning a service to pimple, it should be a creation function that is accessed when the service is requested. Directly assigning the finished object also works, but I think assigning a new creation function might be more "in the spirit" of this DI container. I'll ask on StackOverflow about this.